desktop-src/direct3d9/queries.md

```diff
@@ -152,7 +152,7 @@ From each of the three query states, here are the [**GetData**](/windows/win32/a
 
  
 
-For example, when a query is in the issued state and the answer to the query is not available, [**GetData**](/windows/win32/api/d3d9helper/nf-d3d9helper-idirect3dquery9-getdata) returns S\_FALSE. When the resource finishes its work and the application has issued a query end, the resource transitions the query to the signaled state. From the signaled state, **GetData** returns S\_OK which means that the answer to the query is also returned in *pData*. For instance, here is the sequence of events to return the number of pixels drawn in a render sequence:
+For example, when a query is in the issued state and the answer to the query is not available, [**GetData**](/windows/win32/api/d3d9helper/nf-d3d9helper-idirect3dquery9-getdata) returns S\_FALSE. When the resource finishes its work and the application has issued a query end, the resource transitions the query to the signaled state. From the signaled state, **GetData** returns S\_OK which means that the answer to the query is also returned in *pData*. For instance, here is the sequence of events to return the number of samples shaded in a render sequence:
```

Comment:
Good call, what do you think is the best terminology here to combine truthfulness and "not going into much detail"?

Reply:
I wouldn't use the term MSAA as D3D9 exclusively refers to "multisampling", but otherwise I agree.

Reply:
Yes, and "pixels (or multisampling samples when multisampling is enabled)" would probably be easier to understand since it starts with a simpler concept. While going from general concepts to special cases is natural in specifications where coverage of possible situations is the most important, for learning it's probably the opposite, otherwise we'd be teaching arithmetic in elementary school in the terms commonly used in math papers and starting from complex number matrices of arbitrary dimensionalities or something like that.